Gaussian Elimination/gauss.cpp

```diff
@@ -0,0 +1,174 @@
+#include "stdafx.h"
```

Comment:
Is this available in GCC C++ Compiler / Clang as well? It would be great to have an equivalent GCC library here instead.

Reply:
Removed redundant header and replaced some functions